### DISCUSSION TITLE
Developer tools - states changed in 2022.3.0

### DISCUSSION THREAD

**Discussion by stomko11** [home-assistant/frontend#11922] [Configuration]
I use developer tools - states a lot. I got (partially) used to always needing to expand "set state" section as it is by default collapsed (still it would be great if one could choose, simple ckeckbox somewhere in the corner would do).
However after latest update, when I search for entity (using friendly name), I pick device from the list, it returns me back just friendly name in "entity" field, and not entity ID as it used to. 
I got used to looking up entities there, checking states, and then just copying entity ID. Now that option is gone.
I can search below in "entity" section, however with "garage light" for instance, it will return me all entities that belong to device called "garage light".

![image](https://user-images.githubusercontent.com/88718231/156651039-3de545b9-4f8b-4ee1-8959-6daddfa7fcdc.png)

**Comment by spacegaier** [comment]
Duplicate of https://github.com/home-assistant/frontend/issues/11938
